frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/useProjectActionsForm.ts

```diff
@@ -1,14 +1,21 @@
 import { useActions } from 'hooks/api/getters/useActions/useActions';
-import { IAction, IActionSet } from 'interfaces/action';
+import { IActionSet } from 'interfaces/action';
 import { useEffect, useState } from 'react';
+import { UIAction } from './ActionItem';
 
-enum ErrorField {
+export enum ErrorField {
     NAME = 'name',
     TRIGGER = 'trigger',
     ACTOR = 'actor',
     ACTIONS = 'actions',
 }
 
+export interface IActionFilter {
+    id: string;
+    parameter: string;
+    value: string;
+}
+
```

Comment:
If this belongs here then I would say https://github.com/Unleash/unleash/pull/6115/files#diff-d6d443a71305cc286646ee3e50aeece9f0493f1ea7024016bae7ec848de7a7f0R19 does too.